<h1><strong> Question: </strong></h1>

<h2><em> How many passengers were male? </em></h2>

<h1><strong> Study Description: </strong></h1>

<h2> Census because I'll need to use the entire database to find the total number of males on board. </h2>

<h1><strong> Analysis: </strong></h1>

In [11]:
import pandas as pd

In [12]:
titanic = pd.read_csv("titanic.csv")

In [13]:
male_count = (titanic['Sex'] == "male").sum()

In [14]:
print(male_count)

577


In [15]:
males_count = (titanic[titanic["Sex"] == "male"])

In [16]:
count = len(males_count)
print("Number of males that were on the Titanic:", count)

Number of males that were on the Titanic: 577


In [17]:
titanic.head()

,PassengerId,Survived,Pclass,Name,Sex,Age,SibSp,Parch,Ticket,Fare,Cabin,Embarked
0,1,0,3,"Braund, Mr. Owen Harris",male,22.0,1,0,A/5 21171,7.2500,NaN,S
1,2,1,1,"Cumings, Mrs. John Bradley (Florence Briggs Th...",female,38.0,1,0,PC 17599,71.2833,C85,C
2,3,1,3,"Heikkinen, Miss. Laina",female,26.0,0,0,STON/O2. 3101282,7.9250,NaN,S
3,4,1,1,"Futrelle, Mrs. Jacques Heath (Lily May Peel)",female,35.0,1,0,113803,53.1000,C123,S
4,5,0,3,"Allen, Mr. William Henry",male,35.0,0,0,373450,8.0500,NaN,S


In [18]:
titanic.info()

<class 'pandas.DataFrame'>
RangeIndex: 891 entries, 0 to 890
Data columns (total 12 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   PassengerId  891 non-null    int64  
 1   Survived     891 non-null    int64  
 2   Pclass       891 non-null    int64  
 3   Name         891 non-null    str    
 4   Sex          891 non-null    str    
 5   Age          714 non-null    float64
 6   SibSp        891 non-null    int64  
 7   Parch        891 non-null    int64  
 8   Ticket       891 non-null    str    
 9   Fare         891 non-null    float64
 10  Cabin        204 non-null    str    
 11  Embarked     889 non-null    str    
dtypes: float64(2), int64(5), str(5)
memory usage: 118.9 KB


<h1><strong> Conclusion: </strong></h1>

<h2> The total number of male passengers who were on the Titanic was 577. </h2>